harpy/popgroup.py

```
@@ -25,15 +25,15 @@ def popgroup(inputdir, output):
     """
     try:
         samplenames = set()
-        re_ext = re.compile("\.(bam|sam)$", re.IGNORECASE)
+        re_ext = re.compile(r"\.(bam|sam)$", re.IGNORECASE)
         for i in os.listdir(inputdir):
             if i.lower().endswith(".bam") or i.lower().endswith(".sam"):
                 samplenames.add(re_ext.sub("", os.path.basename(i)))
         if len(samplenames) < 1:
             raise Exception
     except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)